recipes-graphics/wayland/weston_11.0.3.imx.bb

```diff
@@ -162,8 +162,8 @@ DEFAULT_PREFERENCE = "-1"
 SRC_URI:remove = "https://gitlab.freedesktop.org/wayland/weston/uploads/f5648c818fba5432edc3ea63c4db4813/${BPN}-${PV}.tar.xz"
 SRC_URI:prepend = "git://github.com/nxp-imx/weston-imx.git;protocol=https;branch=${SRCBRANCH} "
 SRC_URI += "file://0001-Revert-protocol-no-found-wayland-scanner-with-Yocto-.patch"
-SRCBRANCH = "weston-imx-11.0.2"
-SRCREV = "a3c5e63aa94f8d27b72780e333370be78c9a7653"
+SRCBRANCH = "weston-imx-11.0.3"
+SRCREV = "a509c319b7bfc72638c9b9d8f5e4b6259aba0ab7"
```

Comment:
You also need to update `PREFERRED_VERSION_weston` in `imx-base.inc`.

Reply:
Thank you @thochstein , I missed that :)